### ISSUE TITLE
Nested aggregations not work

### ISSUE THREAD

**Issue by jianbingfang** [danpaz/bodybuilder#28]
The nested aggregations example in your doc seems not work, it only generated the first agg params.

**Comment by nfantone** [comment]
@jianbingfang It seems to be working fine for me. What's the output if you try the query on the REPL?

This is what I see:

```js
bodybuilder > var body = new Bodybuilder().aggregation('terms', 'code', null, { order: { _term: 'desc' }, size: 1 }, agg => agg.aggregation('terms', 'name')).build();
undefined
bodybuilder > JSON.stringify(body);
'{"aggregations":{"agg_terms_code":{"terms":{"field":"code","order":{"_term":"desc"},"size":1},"aggs":{"agg_terms_name":{"terms":{"field":"name"}}}}}}'
```

That is the expected output.

**Comment by jianbingfang** [comment]
Hi @nfantone , thanks for your response, but my result is as below:

`{
  "aggregations": {
    "agg_terms_code": {
      "terms": {
        "field": "code"
      }
    }
  }
}`

My version is bodybuilder@1.2.0, is it same with yours?

**Comment by jianbingfang** [comment]
I think it's due to an old version, and problem had been resolved after updated to v1.8.0.
